extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -43,7 +43,7 @@
      * This property is applicable to reactive REST clients only.
      */
     @ConfigItem(defaultValue = "false")
-    public Optional<Boolean> disableSmartProduces;
+    public Optional<Boolean> disableSmartProduces = Optional.empty();
```

Comment:
is there any point in initializing these fields?

Reply:
I added it so that the fields are not null in certain tests, but probably it will be better to handle that in the test class rather than here in the config root. Will fix.